src/packages/@ncigdc/components/analysis/availableAnalysis.js

```diff
@@ -407,6 +407,7 @@ const availableAnalysis: [TAnalysis] = [
         </p>
       </React.Fragment>
     ),
+    isBeta: true,
```

Comment:
Add this to other analyses to display Beta tag on their selection fields, and result tabs.